From adbffe066fd4141fd65b59b60afe20fc7104b512 Mon Sep 17 00:00:00 2001
From: Pavel Hrdina <phrdina@redhat.com>
Date: Thu, 4 May 2017 13:03:34 +0200
Subject: virtinst.cpu: don't validate "cpus" for NUMA cells

This attribute doesn't set pinning to host CPUs, it configures which
guest vCPUs should belong to that guest NUMA node.  If a guest has
more vCPUs than host you wouldn't be able to configure all guest
vCPUs to the NUMA nodes for guest.

Signed-off-by: Pavel Hrdina <phrdina@redhat.com>
(cherry picked from commit 07eecfdcc780f7482d3636694e9e90bd5f590ce4)

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1281526

Signed-off-by: Pavel Hrdina <phrdina@redhat.com>
---
 virtinst/cpu.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/virtinst/cpu.py b/virtinst/cpu.py
index cd9fa68e..468853f5 100644
--- a/virtinst/cpu.py
+++ b/virtinst/cpu.py
@@ -17,7 +17,6 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 # MA 02110-1301 USA.
 
-from .domainnumatune import DomainNumatune
 from .xmlbuilder import XMLBuilder, XMLProperty, XMLChildProperty
 
 
@@ -28,11 +27,8 @@ class _CPUCell(XMLBuilder):
     _XML_ROOT_NAME = "cell"
     _XML_PROP_ORDER = ["id", "cpus", "memory"]
 
-    def _validate_cpuset(self, val):
-        DomainNumatune.validate_cpuset(self.conn, val)
-
     id = XMLProperty("./@id", is_int=True)
-    cpus = XMLProperty("./@cpus", validate_cb=_validate_cpuset)
+    cpus = XMLProperty("./@cpus")
     memory = XMLProperty("./@memory", is_int=True)
 
 
-- 
2.13.0

